torchgeo/datamodules/chabud.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ChaBuD datamodule."""
+
+from typing import Any
+
+import torch
+from einops import repeat
+
+from ..datasets import ChaBuD
+from .geo import NonGeoDataModule
+
+
+class ChaBuDDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the ChaBuD dataset.
+
+    Uses the train/val splits from the dataset
```

Comment:
No test split? Should we make one by subdividing val?

Reply:
I'll have to check back in on the competition. I'm not sure if they made a test set public or if it just has no labels.

Reply:
I'll add a test set in a new PR. Looks like they changed the format of the test file so it will take some significant changes.